SetupDataPkg/Docs/PlatformIntegration/PlatformIntegrationSteps.md

```diff
@@ -82,6 +80,8 @@ Following these two operations, the n number of Config Policy to Silicon Policy
 policy and override the relevant silicon policies. An example is provided in
 [mu_tiano_platforms](https://github.com/microsoft/mu_tiano_platforms/blob/HEAD/Platforms/QemuQ35Pkg/ConfigKnobs/ConfigKnobs.c).
 
+Note that, for demonstration simplicity and data overhead reduction, the provided example above directly intialized the silicon policy and updaate the value using data from configuration autogen headers. In real world, this same model could also be considered as long as this model would not cause dependency intervention.

```

Comment:
Need to note that this could also be used on alternatives of "policy service". Intel setup data, structured PCD, etc.

Reply:
Done.